src/cpu/minor/dyn_inst.cc

```diff
@@ -112,6 +112,11 @@ MinorDynInst::reportData(std::ostream &os) const
 std::ostream &
 operator <<(std::ostream &os, const MinorDynInst &inst)
 {
+    if (!inst.pc) {
```

Comment:
Are there cases where there isn't a pc associated with a dynamic instruction but still a fault or something else we want to print? I can't think of anything obvious, but you could imagine some sort of synthetic dynamic instruction going through the pipeline to raise an interrupt or something similar.

Reply:
If there is no PC, then it prints `0x????????`. If, in the future, someone wants to change this, it should be easy to track it down.

This is a clear improvement over the current code which generates a segfault if there is no PC.

Let me know if you have a specific suggestion for improvement.